modules/UI/videolayout/VideoLayout.js

```diff
@@ -663,14 +662,6 @@ const VideoLayout = {
             largeVideo.updateContainerSize();
             largeVideo.resize(animate);
         }
-
```

Comment:
what was this upposed to do before? looks like dead code... or did it rely onside-effects?

Reply:
No idea :) I just saw it is not doing anything meaningful and deleted it...

I suppose during some refactoring somebody forgot to remove it...

Reply:
Sweet code deletion :-)